From 75e0fe28bb02036dd404eec53dc3b71b55684ed1 Mon Sep 17 00:00:00 2001
From: Arnaldo Carvalho de Melo <acme@redhat.com>
Date: Mon, 10 Oct 2022 16:59:53 -0300
Subject: [PATCH 14/55] core: Add DW_TAG_unspecified_type to tag__is_tag_type()
 set

It is a type, so make tag__is_tag_type() return true for it.

Signed-off-by: Arnaldo Carvalho de Melo <acme@redhat.com>
---
 dwarves.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dwarves.h b/dwarves.h
index d1aa240..9aaf87f 100644
--- a/dwarves.h
+++ b/dwarves.h
@@ -547,6 +547,7 @@ static inline int tag__is_tag_type(const struct tag *tag)
 	       tag->tag == DW_TAG_unspecified_type ||
 	       tag->tag == DW_TAG_volatile_type ||
 	       tag->tag == DW_TAG_atomic_type ||
+	       tag->tag == DW_TAG_unspecified_type ||
 	       tag->tag == DW_TAG_LLVM_annotation;
 }
 
-- 
2.40.0.rc2

